opendbc/car/__init__.py

```diff
@@ -7,14 +8,24 @@
 from opendbc.car import structs, uds
 from opendbc.car.can_definitions import CanData
 from opendbc.car.docs_definitions import CarDocs, ExtraCarDocs
+from opendbc.car.vehicle_model import VehicleModel
 
 DT_CTRL = 0.01  # car state and control loop timestep (s)
 
 # kg of standard extra cargo to count for drive, gas, etc...
 STD_CARGO_KG = 136.
 
+# ISO 11270
+ISO_LATERAL_ACCEL = 3.0  # m/s^2
+ISO_LATERAL_JERK = 5.0  # m/s^3
+
 ACCELERATION_DUE_TO_GRAVITY = 9.81  # m/s^2
 
+# Add extra tolerance for average banked road since safety doesn't have the roll
+AVERAGE_ROAD_ROLL = 0.06  # ~3.4 degrees, 6% superelevation. higher actual roll lowers lateral acceleration
+MAX_LATERAL_ACCEL = ISO_LATERAL_ACCEL + (ACCELERATION_DUE_TO_GRAVITY * AVERAGE_ROAD_ROLL)  # ~3.6 m/s^2
```

Comment:
This shouldn't be a global in /car/__init__.py. It can be very confusing for users to know what to import/use

Reply:
It's important to scope things to the minimum required level until we need otherwise

Reply:
moving this to `lateral.py` feels like a more sensible approach, do you agree @sshane ?